### ISSUE TITLE
journald manual does not describe built-in upper limit

### ISSUE THREAD

**Issue by mischief** [systemd/systemd#1441]
in the journald.conf manual, it claims that 10% of a filesystem's space will be used by default.

however, in src/journal/journal-file.c we see:

``` c
                if (fs_size > 0) {
                        m->max_use = PAGE_ALIGN(fs_size / 10); /* 10% of file system size */

                        if (m->max_use > DEFAULT_MAX_USE_UPPER)
                                m->max_use = DEFAULT_MAX_USE_UPPER;

```

here max_use is set to DEFAULT_MAX_USE_UPPER, which is 4G, and could be less than the advertised '10% of filesystem space'.

**Comment by poettering** [comment]
Fix is in #1452.

**Comment by mischief** [comment]
thanks.
